### ISSUE TITLE
bug: accessibility failure with ion-button and ion-icon

### ISSUE THREAD

**Issue by mhartington** [ionic-team/ionic-framework#18272]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**

When running an app through lighthouse, the accessibility audit will point out ion-button and ion-icon for failing to have the proper aria-labels and alt attributes.

<img width="1260" alt="Screen Shot 2019-05-13 at 6 42 01 PM" src="https://user-images.githubusercontent.com/2835826/57659018-d3390700-75ae-11e9-9c36-c9282125614b.png">

- https://dequeuniversity.com/rules/axe/3.1/button-name?application=lighthouse
- https://dequeuniversity.com/rules/axe/3.1/image-alt?application=lighthouse

**Comment by amymarsh4work** [comment]
I just tested 4.7.1 using the forked ionic project, running 'start' in Chrome and Firefox. I tested 4 of the core/button/test pages: basic, dynamic, form and icon using these extensions: Chrome Lighthouse, aXe and WAVE - the specific problem described in this issue no longer exists. :)

**Comment by Jannnnnn** [comment]
I still have a warning at 4.7.1 and the ion-toggle in the Lighthouse's audits

**Comment by brandyscarney** [comment]
Confirmed this is no longer an issue on latest.
